docs/ai/conceptual/azure-ai-services-authentication.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Authenticate to Azure OpenAI using .NET
+description: Learn about the different options to authenticate to Azure OpenAI and other services using .NET
+author: alexwolfmsft
+ms.topic: conceptual
+ms.date: 06/27/2024
+---
+
+# Authenticate to Azure AI Services using .NET
```

Comment:
I wonder if the title should include the word "authorization" too. I say that because RBAC is covered in the doc.